static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
When we tried to require something using the `require` that was previously manually imported, e.g. `require(['foo'])`, the `foo` module ID was not ran through `System.normalize`.

When we removed the manual require of `require`, thus relying on `window.require = System.amdRequire`, all was fine.